anyTV/music-dashboard
# Fix/tunes 140

## A. Summary of Changes

### a. DB Changes:
- N/A

### b. Package Changes:
- none

### c. Implementation Changes:
- Fixed flickering audio player seek bar

## B Reference:

### a. Ticket link/Request Feature link
- [Tunes-140](https://freedom.myjetbrains.com/youtrack/issue/Tunes-140)

### b. Tests Done
- Play a track
- Open playlist
- Seek bar should not flicker
- Close the playlist
- Pause the track
- Seek bar should not change
- http://moztrap.freedom.tm/runtests/run/98/env/91/?pagenumber=1&pagesize=20&sortfield=order&sortdirection=asc&filter-suite=48
